### PR TITLE
Include pending versions in 'Reject multiple versions' action

### LINKED ISSUE #7968: 'Reject multiple versions' does not include version pending review
When rejecting multiple versions in the reviewer tools, a version that is currently pending review (not auto-approved) is not rejected.

STR:
1. Submit an add-on version. Wait for auto-approval to run.
2. As an admin reviewer, disable auto-approval (yellow button at the bottom of the page).
3. As the developer, submit another version.
4. As the reviewer, try to reject all versions using "Reject multiple versions"

Expected results:

Both the first (auto-approved) and the second (not auto-approved) versions should show up in the list of versions to be rejected.

Actual results:

The second (not auto-approved) version is not in the list of versions to be rejected.

### PR DESCRIPTION
**Pull request by wagnerand**
Fix #7968.

### CODE REVIEW

**Comment by wagnerand** [comment]
PR 8000. 🎉

File: src/olympia/reviewers/forms.py [hunk 1]
```diff
@@ -344,7 +344,8 @@ def __init__(self, *args, **kw):
             self.fields['versions'].queryset = (
                 self.helper.addon.versions.distinct().filter(
                     channel=amo.RELEASE_CHANNEL_LISTED,
-                    files__status=amo.STATUS_PUBLIC).order_by('created'))
+                    files__status__in=(amo.STATUS_PUBLIC, amo.STATUS_PENDING)).
```
Comment (eviljeff) [inline src/olympia/reviewers/forms.py]: `STATUS_PENDING` is for LWT

Comment (wagnerand) [reply src/olympia/reviewers/forms.py]: Uhm, right, but why is the test succeeding anyway?

File: src/olympia/reviewers/tests/test_forms.py [hunk 2]
```diff
@@ -151,6 +151,25 @@ def test_versions_queryset(self):
         assert list(form.fields['versions'].queryset) == [
             self.addon.current_version]
 
+    def test_versions_queryset_contains_pending_version(self):
+        addon_factory()
+        file_factory(version=self.addon.current_version,
+                     status=amo.STATUS_AWAITING_REVIEW)
```
Comment (eviljeff) [inline src/olympia/reviewers/tests/test_forms.py]: this adds an extra file to `current_version` (which would contain a single file with status `STATUS_PUBLIC` already) so you're not changing the number of versions with valid files shown.

File: src/olympia/reviewers/utils.py [hunk 3]
```diff
@@ -357,7 +357,7 @@ def get_actions(self, request):
             self.version.channel == amo.RELEASE_CHANNEL_UNLISTED and
             is_unlisted_reviewer)
         is_public_and_listed_and_user_can_post_review = (
-            self. version and
+            self.version and
```
Comment (wagnerand) [inline src/olympia/reviewers/utils.py]: Bonus!

Comment (eviljeff) [reply src/olympia/reviewers/utils.py]: what the...?

**Pull request merged**